src/components/Footer.js

```diff
@@ -3,8 +3,12 @@ import React, { Component } from 'react'
 class Footer extends Component {
   render () {
     return (
-      <div>Footer</div>
-      
+     <footer>
+      <div>
+        <div class="header"><img src="https://www.clipartmax.com/png/full/146-1467402_clipart-info-peace-love-weed.png" height="100" width="auto" />      
+        </div>
+      </div>
+      </footer>
```

Comment:
Hey it looks like this isn't indented correctly please fix that up and push up your changes and I'll approve.

Reply:
took a look, fixed. 100%